src/app/features/profile/email-verification/email-verification.page.ts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@angular/core';
+import { TranslocoService } from '@ngneat/transloco';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { defer, timer } from 'rxjs';
+import { catchError, first, map, switchMap, take, tap } from 'rxjs/operators';
+import { BlockingActionService } from '../../../shared/blocking-action/blocking-action.service';
+import { DiaBackendAuthService } from '../../../shared/dia-backend/auth/dia-backend-auth.service';
+import { ErrorService } from '../../../shared/error/error.service';
+
+@UntilDestroy({ checkProperties: true })
+@Component({
+  selector: 'app-email-verification',
+  templateUrl: './email-verification.page.html',
+  styleUrls: ['./email-verification.page.scss'],
+})
+export class EmailVerificationPage {
+  hasSentEmailVerification = false;
+  secondsRemained = 0;
+  readonly RESEND_INTERVAL: number = 60;
+
+  readonly emailVerified$ = this.diaBackendAuthService.emailVerified$;
+  readonly email$ = this.diaBackendAuthService.email$;
+
+  constructor(
+    private readonly blockingActionService: BlockingActionService,
+    private readonly diaBackendAuthService: DiaBackendAuthService,
+    private readonly errorService: ErrorService,
+    private readonly translocoService: TranslocoService
+  ) {}
+
+  async sendEmailVerification() {
+    const ONE_SECOND = 1000;
+    const action$ = defer(() => this.email$).pipe(
+      first(),
+      switchMap(email =>
```

Comment:
When there's no need to cancel previous inner observable (`resendActivationEmail$`) when source observable (`email$`) emits new value, use `concatMap` instead.

It's nearly effectively the same in this case, but when not intentionally want to use the cancel capability of `switchMap`, `concatMap` is a more natural choice.